src/data/blog/v-vvv-e-eee.md

```diff
@@ -0,0 +1,53 @@
+---
+title: 'Value and effort and prioritisation.'
+slug: 'v-vvv-e-eee'
+image: null
+featured: 0
+page: 0
+status: 'published'
+language: 'en_US'
+meta_title: null
+meta_description: null
+author_id: 5
+created_at: '2020-01-30T21:26:07.000Z'
+created_by: 5
+updated_at: '2020-01-30T09:59:00.000Z'
+updated_by: 1
+published_at: '2020-01-30T16:12:19.000Z'
+published_by: 5
+visibility: 'public'
+mobiledoc: null
+amp: null
+author_name: 'Harry Harrold'
+author_slug: 'harry-harrold'
+author_image: '//www.gravatar.com/avatar/febf032ab16fef166824f5172cd87393?s=250&d=mm&r=x'
+author_cover: null
+author_facebook: null
+author_twitter: null
+---
+
+Every sprint, Neontribe’s developers are faced with a stack of work, in an order. Every sprint, we start with work from the top of the stack, and carry on for as long as we have planned
+
+We order this stack using Vs and Es; rough of the Value and Effort of a piece of work. Normally, these pieces of work at least start as user stories; value we are delivering to a user of the software we’re creating. Every piece of work will start with one to three Vs and one to three Es.
```

Comment:
`; rough of the Value` is missing some words I think.

Reply:
Thank you! Replacing with "rough ideas of the Value and Effort of a piece of work."

Reply:
Still says "rough of the Value and Effort"